feature/platform/transceiver/tests/zr_pm_test/README.md

```diff
@@ -0,0 +1,89 @@
+# TRANSCEIVER-6: Telemetry: 400ZR Optics pm streaming.
+
+## Summary
+
+Validate 400ZR optics module reports performance metric (PM) data as defined in
+module CMIS VDM(Versatile Diagnostics Monitor):
+*   eSNR is defined as the electrical Signal to Noise ratio at the decision sampling point in dB
+*   Q-value is the decibel (dB) value representing signal BER.
+*   pre-FEC BER bit error rate.
```

Comment:
This test appears to be combining portions of 3 tests together into one and is renaming the tests.    @ahsaanyousaf is this what you want to do?  I thought the breakout was rather nice before since it makes it easier to see which paths/subfeatures are working or not working.  But if there is some good reason to combine them, please let us know.

This test partially overlaps with : 
`TRANSCEIVER-1: Telemetry: 400ZR Electrical Signal to Noise Ratio(eSNR) and Chromatic Dispersion(CD) telemetry values streaming`


* TRANSCEIVER-2: Telemetry: 400ZR Optics Pre-FEC(Forward Error Correction) BER(Bit Error Rate)
* TRANSCEIVER-6: Telemetry: 400ZR Optics Q-value streaming.

If we want to do this combining, then the affected readmes should be deleted or modified appropriately and the [testregistry.textproto](https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto) updated as well.

Reply:
I heard about some concerns related to testbed reservation and execution time as we have to reserve, configure and tier down the config setup each time so in favor of reserving the testbeds just once for all the relevant telemetry leaves under /terminal-device/logical-channels/channel/otn/state/ we decided to combine these PMs in a single test. @dplore please let us know if that will be a problem.

If this is good Reza can update the related READMEs to comply to whats covered under this test

Reply:
This is is ok with me and is a reasonable optimization (it doesn't make this test too big).  This PR should include changing/removing the affected README's and testregistry.textproto.

Reply:
Done. Deleted the tests that are added to pm test and updated the testregistery proto.